CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
+### Added
+- added Content-type-mismatch-to-reponse-body.php for testing new passive rule 1204704.
```

Comment:
@itsdivyanshjain  can u please add the name of the rule here rather than alertId

Reply:
Will be releasing new version update with this patch, so can u please add ` [0.0.3] - 2023-06-19` also. 😄